CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+## Alfalfa Version 0.1.0
```

Comment:
Ok so this is the version of record right? the main place the version is recorded and tracked.